jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/ContextFactory.java

```diff
@@ -28,32 +28,37 @@
 import com.ibm.dtfj.java.JavaRuntime;
 
 /**
- * Factory for creating different types of context
- * 
- * @author adam
+ * Factory for creating different types of contexts.
  *
+ * @author adam
  */
 public class ContextFactory {
-	
+
+	private ContextFactory() {
+		/* no instances needed */
+	}
```

Comment:
This class was internal until #17064; the class only has static features so instances are not required.